### PR TITLE
Add missing GTFS docs

### PR DESCRIPTION
**Pull request by barbeau**

### CODE REVIEW

**Comment by barbeau** [comment]
Actually, hold off on reviewing for a moment - just realized that another Readme and other pages are missing.  Will add those shortly.

**Comment by barbeau** [comment]
@bboissin Ok, just forced-pushed a new commit into this PR to replace the last one, so it's ready for review.

Here's the summary (also in commit description):

* Add community.md from https://developers.google.com/transit/community
* Add style-guide.md from https://developers.google.com/transit/gtfs/style-guide
* Add tools.md from https://developers.google.com/transit/tools
* Move CHANGES.md to mirror GTFS-realtime CHANGES.md directory structure, update change process to match GTFS-realtime (i.e., via Github PRs)
* Add README to root GTFS directory with link to reference and changes doc

File: gtfs/CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  - Votes against shall be motivated, and ideally provide actionable feedback.
+  - If the vote has failed, then the advocate may choose to continue work on the proposal, or to abandon the proposal.
+    Either decision of the advocate must be announced in the mailing list.
+  - If the advocate continues the work on proposal then a new vote can be called for at any point in time but no later than 30 calendar days after the end of the previous vote.
+  - If a vote was not called within 30 calendar days from the original proposal or 30 calendar days since end of the previous vote, then the proposal is abandoned.
+1. If the proposal is abandoned, the corresponding pull request is closed.
+1. If the proposal is accepted:
+  - Google is committed to merging the voted upon version of the pull request (provided that the contributors have signed the [CLA](../CONTRIBUTING.md)), and performing the pull request within 5 business days.
+  - Google is committed to timely updating https://github.com/google/gtfs-realtime-bindings repository. Commits to gtfs-realtime-bindigs that are a result of a proposal, should reference the pull request of the proposal.
```
Comment (egorich239) [inline gtfs/CHANGES.md]: Remove this bullet? There's nothing specific to GTFS in the referred repo, right?

File: gtfs/CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  - Translations must not be included into the original pull request.
+    Google is responsible for eventually updating relevant translations into supported languages, but pure translation pull requests from the community are welcome and will be accepted as soon as all editorial comments are addressed.
+
+### Guiding Principles
+In order to preserve the original vision of GTFS, a number of guiding principles have been established to take into consideration when extending the spec:
+
+#### Feeds should be easy to create and edit.
```
Comment (egorich239) [inline gtfs/CHANGES.md]: The above headers didn't have full stops. Mabe these also don't?

File: gtfs/CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Changes to the spec should be backwards-compatible.
+When adding features to the specification, we want to avoid making changes that will make existing feeds invalid. We don't want to create more work for existing feed publishers until they want to add capabilities to their feeds. Also, whenever possible, we want existing parsers to be able to continue to read the older parts of newer feeds.
+
+#### Speculative features are discouraged.
+Every new feature adds complexity to the creation and reading of feeds. Therefore, we want to take care to only add features that we know to be useful. Ideally, any proposal will have been tested by generating data for a real transit system that uses the new feature and writing software to read and display it. Note that the GTFS readily allows for extensions to the format through the addition of extra columns and files that are ignored by the official parsers & validators, so proposals can be easily prototyped and tested on existing feeds.
+
+### Revision History
+
+#### January 13, 2016
```
Comment (barbeau) [inline gtfs/CHANGES.md]: Just caught this - I'll update to the newer date.

File: gtfs/CHANGES.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Changes to the spec should be backwards-compatible.
+When adding features to the specification, we want to avoid making changes that will make existing feeds invalid. We don't want to create more work for existing feed publishers until they want to add capabilities to their feeds. Also, whenever possible, we want existing parsers to be able to continue to read the older parts of newer feeds.
+
+#### Speculative features are discouraged.
+Every new feature adds complexity to the creation and reading of feeds. Therefore, we want to take care to only add features that we know to be useful. Ideally, any proposal will have been tested by generating data for a real transit system that uses the new feature and writing software to read and display it. Note that the GTFS readily allows for extensions to the format through the addition of extra columns and files that are ignored by the official parsers & validators, so proposals can be easily prototyped and tested on existing feeds.
+
+### Revision History
```
Comment (egorich239) [inline gtfs/CHANGES.md]: Early Revision History?

Or do we want to track anything on top of Github history?

File: gtfs/CHANGES.md (hunk 1 above)
Comment (barbeau) [inline gtfs/CHANGES.md]: Ah, right, good catch.  Will remove it.

File: gtfs/CHANGES.md (hunk 4 above)
Comment (barbeau) [inline gtfs/CHANGES.md]: I say we keep the Revision History here.  It's nice to have a clean list of changes to the spec without some of the noise in Git commit histories.  And, assuming this all still gets copied over to http://developers.google.com/transit/gtfs/ we need a readable version for HTML, especially for those less technically inclined that aren't visiting Github.

File: gtfs/CHANGES.md (hunk 2 above)
Comment (barbeau) [inline gtfs/CHANGES.md]: Ok, I'll remove the periods from the subheadings.

**Comment by barbeau** [comment]
@egorich239 @bboissin Ok, I've force-pushed a new commit here to replace the old one to address the comments.  As commented above on the previous diff, I would prefer to keep the plain text version of the Revision History.  Please let me know if you think otherwise.

File: gtfs/CHANGES.md (hunk 4 above)
Comment (egorich239) [inline gtfs/CHANGES.md]: sound good to me

**Pull request merged**